src/core/Header/HeaderTitle.tsx

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+
+import cx from 'classnames';
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+
+export type HeaderTitleProps = {
+  /**
+   * Logo shown before the main content.
+   */
+  logo: JSX.Element;
+} & React.ButtonHTMLAttributes<HTMLDivElement>;
+
+/**
+ * Header Title section
+ * @example
+ * <HeaderTitle logo="<SvgMicrostation />">Microstation</HeaderTitle>
+ * <HeaderTitle logo="<SvgItwin />" />
```

Comment:
Maybe use real examples (we dont have these svgs in the package) of svgs and possibly one for png too

Reply:
I can use <SvgPlaceholder> or <svg ... />, is there a preference ? (Not sure its worth having the full content of an svg here.)

Reply:
Any `Svg-` that's actually valid. So I guess use `SvgImodelHollow`